pommerman/network/ionc/__init__.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+"""IonClient Entry point"""
+
+# Global Imports
+import ui
+from .constants import LangEN as Strings
+from .network import Network as net
+from signal import signal, SIGINT
+from gym.spaces import Discrete
+
+
+# Functions
+def init():
+    """Initialize the application"""
+
+    def exit_handler(_s, _h):
+        """Handle exiting the application"""
+        ui.fatal(ui.yellow, "Exiting..")
+
+    signal(SIGINT, exit_handler)
+    if ui.ask_yes_no(Strings.server_prompt.value):
+        domain = ui.ask_string(Strings.server_domain.value)
+        if domain is None:
+            ui.fatal(Strings.e_invalid_domain.value)
```

Comment:
Does this exit everywhere or only this thread or ...?

Reply:
The client only runs on one thread even though the network class supports usage of multiple threads.